### PR TITLE
Added files for Lansweeper App For Sentinel Custom Connector

### PR DESCRIPTION
**Pull request by jayeshprajapaticrest**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by natalie-pienkowska** [comment]
Hi @jayeshprajapaticrest! Our certification engineer, Srikanth, is currently on emergency leave. Please expect a delay with your PR. He will return January 31st, 2022. We apologize for any inconveniences.

File: certified-connectors/Lansweeper App For Sentinel/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        ],
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "description": "Content-Type",
+			"x-ms-summary": "Content-Type",
```
[the remark this answers is not in the document]
Comment (jayeshprajapaticrest) [reply certified-connectors/Lansweeper App For Sentinel/apiDefinition.swagger.json]: @sriyen-msft 
We have removed extra spaces from required places and keep the swagger formatted.

File: certified-connectors/Lansweeper App For Sentinel/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                "data": {
+                  "type": "object",
+                  "properties": {},
+                  "description": "data"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request"
```
[the remark this answers is not in the document]
Comment (jayeshprajapaticrest) [reply certified-connectors/Lansweeper App For Sentinel/apiDefinition.swagger.json]: **### @sriyen-msft** 
We have added possible schema for the 400 response.
But after doing that during Validation part using paconn command it gives warning as below.

"Swagger certification succeeded with warnings. It is not required but recommended to fix following warnings: Warning : paths/~1api~1v2~1graphql/post/responses : The operation 'RetrieveAssetDetails' has more than one response with specified schema, only one schema will be used in the designer."

File: certified-connectors/Lansweeper App For Sentinel/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+          "tooltip": "Please provide the API Token",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```
[the remark this answers is not in the document]
Comment (jayeshprajapaticrest) [reply certified-connectors/Lansweeper App For Sentinel/apiProperties.json]: **@sriyen-msft**
We have used icon which doesn't contain the Invalid content as per submission requirement.
And updated brand color of our icon in file.

File: certified-connectors/Lansweeper App For Sentinel/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Lansweeper App For Sentinel
```
[the remark this answers is not in the document]
Comment (jayeshprajapaticrest) [reply certified-connectors/Lansweeper App For Sentinel/README.md:1]: @sriyen-msft 
We have updated readme.md file which include all the possible description according to the provided templates.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**